tests/keras/backend/backend_test.py

```diff
@@ -1064,6 +1049,20 @@ def test_conv(self, op, input_shape, kernel_shape, padding, data_format):
             padding=padding, data_format=data_format,
             cntk_dynamicity=True)
 
+    @pytest.mark.parametrize(
+        'op,input_shape,kernel_shape,output_shape,padding,data_format', [
+            ('conv2d_transpose', (2, 5, 6, 3), (3, 3, 2, 3), (2, 5, 6, 2),
+             'same', 'channels_last'),
+            ('conv2d_transpose', (2, 3, 8, 9), (3, 3, 2, 3), (2, 2, 8, 9),
+             'same', 'channels_first'),
+        ])
+    def test_conv_transpose(self, op, input_shape, kernel_shape, output_shape,
+                            padding, data_format):
+        check_two_tensor_operation(
+            op, input_shape, kernel_shape, WITH_NP,
+            output_shape=output_shape,padding=padding, data_format=data_format,
```

Comment:
There is a space missing before `padding`. I'm surprised that Travis didn't catch that when running the pep8 tests.

Reply:
It seems that all pep8 tests failures are ignored by travis. I'll make a PR to fix it.

Reply:
Updated. Thank you for the comment.